service/client/client.js

```diff
@@ -38,6 +38,10 @@ export default class Client {
         this.token = token;
     }
 
+    getUrlVersion() {
```

Comment:
it's called here https://github.com/mattermost/mattermost-mobile/pull/144/files/c1b750df0b16d0008ac77a808d8b0eb032c6fda7#diff-b599a7270243eae4109fc315880b40a2R41